tests/unit/spanner_dbapi/test_cursor.py

```diff
@@ -956,7 +963,12 @@ def test_ddls_with_semicolon(self):
         cursor.execute(
             "CREATE TABLE table_name (row_id INT64) PRIMARY KEY ();"
             "DROP INDEX index_name;\n"
-            "DROP TABLE table_name;"
+            "CREATE TABLE papers ("
+            "\n    id INT64,"
+            "\n    authors ARRAY<STRING(100)>,"
+            '\n    author_list STRING(MAX) AS (ARRAY_TO_STRING(authors, ";")) stored'
+            ") PRIMARY KEY (id);"
+            "DROP TABLE table_name;",
```

Comment:
The only difference `sqlparse.split()` brings is that it keeps `;` symbols at the end of statements.

Reply:
That is correct. Spanner expects the DDL statements to not have a `;` character. A simple check can be added after the null check like so:
```
if ddl:
    if ddl[-1] == ";":
        ddl = ddl[:-1]
    self.connection._ddl_statements.append(ddl)
```